datafusion/core/tests/user_defined/user_defined_plan.rs

```diff
@@ -796,22 +798,30 @@ fn accumulate_batch(
     k: &usize,
 ) -> BTreeMap<i64, String> {
     let num_rows = input_batch.num_rows();
+
     // Assuming the input columns are
-    // column[0]: customer_id / UTF8
+    // column[0]: customer_id / UTF8 or UTF8View
```

Comment:
I feel like the example would be simpler if it just used `Utf8View`. Could we do that and avoid adding the logic to handle both types?

Reply:
Good suggestion! Thank you @alamb , addressed in latest PR.